version.txt

```diff
@@ -1 +1 @@
-2.0
\ No newline at end of file
+2.1.0
```

Comment:
We need to pop this to 3.0.0
We already have a 2 series and this is a breakingish change